### PR TITLE
[TRIVIAL] [PDF] Fix table Overloadable Index Unary Operators

### PR DESCRIPTION
**Pull request by andralex**
before: 
a

### CODE REVIEW

**Comment by andralex** [comment]
after: 
b

File: latex.ddoc [hunk 1]
```diff
@@ -260,6 +260,7 @@ SINGLEQUOTE=`$0'
 SMALL=\small{$0}
 SPEC_S=$(LAYOUT , $1, $(ARGS $+))
 SUB=\textsubscript{$0}
+SUBSCRIPT=\textsubscript{$0}
```
Comment (wilzbach) [inline latex.ddoc]: This is exactly similar to the existing, unused `SUB` macro. Which one do you prefer? (let's remove the other)

**Comment by wilzbach** [comment]
(found a matching Bugzilla issue again)

File: latex.ddoc (hunk 1 above)
Comment (andralex) [reply latex.ddoc]: SUB is already "subset", so it should be deleted. I'll do this in a moment.

**Pull request merged**